src/main/java/com/bae/service/PlayerService.java

```diff
@@ -0,0 +1,68 @@
+package com.bae.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.bae.persistence.domain.Player;
+import com.bae.persistence.repo.PlayerRepository;
+
+import exceptions.MemberNotFoundException;
+
+@Service
+public class PlayerService {
+
+	private PlayerRepository playerRepo;
+
+	
+	@Autowired
+	public PlayerService(PlayerRepository playerRepo) {
+		this.playerRepo = playerRepo;
+	}
+
+	// Creating Members
```

Comment:
Do you need these comments? They might make your code lose a few marks (unsure though just a suggestion)